tests/foreman/ui/test_environment.py

```diff
@@ -51,6 +52,40 @@ def test_positive_create_with_name(self):
 
     @run_only_on('sat')
     @tier1
+    def test_positive_create_with_org(self):
+        """Create new environment with organization
```

Comment:
perhaps create_with_org_name and in the future we can create other similar function create_with_org_id

Reply:
sorry,  this is UI test  :)